brambling/views/organizer.py

```diff
@@ -1249,10 +1299,12 @@ def get_forms(self):
                         form.data = self.request.POST
                         form.is_bound = True
                         break
+        self.transaction_forms = [TransactionRefundForm(t)
+                                  for t in self.order.transactions.all()]
         if show_payment_form:
```

Comment:
show_payment_form is a marker of whether the user has permission to edit the event. So we should probably rename the var and only display the transaction forms if it's True as well.